spark/v4.0/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -53,16 +56,20 @@
 
 class AddFilesProcedure extends BaseProcedure {
 
+  public static final String NAME = "add_files";
```

Comment:
not need public (for all these?)  Avoid having too many unnecessary dependency if we need to move these?

Reply:
`SparkProcedures` access it, can be narrowed to `package` level